### PR TITLE
Andrew branch

### PR DESCRIPTION
**Pull request by suyangsong**
added the H1 and stuff

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<html lang="en">
```
Comment (durph) [inline index.html]: Readd this line

**Comment by suyangsong** [comment]
screw u mang I ain't changing shiieeeeett

File: index.html (hunk 1 above)
Comment (suyangsong) [reply index.html]: jk I did it

Comment (kapadiavasim) [reply index.html]: durph durph durph

**Pull request merged**